### DISCUSSION TITLE
Add <input type="numeric"> to list of recognized Input styles

### DISCUSSION THREAD

**Discussion by prostko** [tailwindlabs/tailwindcss#10306] [Ideas]
Inputs have a base style applied to them: 
```
[type='text'], [type='email'], [type='url'], [type='password'], [type='number'], [type='date'], [type='datetime-local'], [type='month'], [type='search'], [type='tel'], [type='time'], [type='week'], [multiple], textarea, select {
    -webkit-appearance: none; -moz-appearance: none;
    appearance: none; background-color: #fff;
    border-color: #6b7280; border-width: 1px;
    border-radius: 0px; padding-top: 0.5rem;
    padding-right: 0.75rem; padding-bottom: 0.5rem;
    padding-left: 0.75rem; font-size: 1rem;
    line-height: 1.5rem;--tw-shadow: 0 0 #0000;
}
```

But obviously "numeric" is absent.  Any objections to adding it? 

Thanks for the project y'all we're loving it.

**Comment by wongjn** [comment]
An objection could be that `numeric` does not seem to be a valid value for the `type` attribute on `<input>` elements – I could not see it in the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#input_types) or the [HTML living standard](https://html.spec.whatwg.org/multipage/input.html).

**Comment by 52461** [comment]
Hello,

[UMR Provider Portal](https://www.umrproviderportal.org/)
.number-wrapper {
  position: relative;
}

.number-wrapper:after,
.number-wrapper:before {
  position: absolute;
  right: 5px;
  width: 1.6em;
  height: .9em;
  font-size: 10px;
  pointer-events: none;
  background: #fff;
}

.number-wrapper:after {
  color: blue;
  content: "\25B2";
  margin-top: 1px;
}

.number-wrapper:before {
  color: red;
  content: "\25BC";
  margin-bottom: 5px;
  bottom: -.5em;
}
